src/modules/app.js

```diff
@@ -0,0 +1,36 @@
+import { createGame, getScores, submitScore } from './game.js';
+
+const refreshButton = document.querySelector("button[type='reset']");
+const submitForm = document.querySelector('form');
+const nameInput = document.getElementById('name');
+const scoreInput = document.getElementById('score');
+const tableBody = document.querySelector('tbody');
+
+refreshButton.addEventListener('click', async () => {
+  const scores = await getScores();
+  tableBody.innerHTML = '';
+  scores.forEach((score) => {
+    const row = document.createElement('tr');
+    const name = document.createElement('th');
+    name.setAttribute('scope', 'row');
+    name.textContent = `${score.user}: ${score.score}`;
+    row.appendChild(name);
+    tableBody.appendChild(row);
+  });
+});
+
+submitForm.addEventListener('submit', async (event) => {
+  event.preventDefault();
+  const name = nameInput.value.trim();
+  const score = scoreInput.value.trim();
+  if (!name || !score) {
+    alert('Please enter both name and score!');
+    return;
+  }
+  const result = await submitScore(name, score);
+  alert(result);
```

Comment:
- [ ]  It will be great to remove this window alert in your application. It’s ugly and impossible to style, it stops code execution and displays differently on different browsers. Use a custom modal instead.

Reply:
well resolved! ✔️